src/functions/sendCandidateResults/application/service/__tests__/terminated-test.cat-be.spec.ts

```diff
@@ -0,0 +1,205 @@
+    });
+
+    it('should send an email for terminated activity code 4', async (done) => {
+      const notifyClient: INotifyClient = new NotifyClientStubSuccess();
+      testResults.forEach(result => result.activityCode = '4');
+      const requestScheduler =
+        new RequestScheduler(configAdapter, notifyClient, templateIdProvider, personalisationProvider, statusUpdater);
+
+      spyOn(notifyClient, 'sendEmail');
+      await requestScheduler.scheduleRequests(testResults);
+
+      setTimeout(() => {
+        expect(notifyClient.sendEmail).toHaveBeenCalled();
+        expect(statusUpdater.updateStatus).toHaveBeenCalledWith({
+          applicationReference: 12345672011,
+          outcomePayload: {
+            interface: NOTIFY_INTERFACE,
+            state: ProcessingStatus.ACCEPTED,
+            staff_number: '123456',
+            retry_count: 0,
+            error_message: null,
+          },
+        });
+        done();
+      },         1000);
+    });
+
+    it('should send an email for terminated activity code 5', async (done) => {
+      const notifyClient: INotifyClient = new NotifyClientStubSuccess();
+      testResults.forEach(result => result.activityCode = '5');
+      const requestScheduler =
+        new RequestScheduler(configAdapter, notifyClient, templateIdProvider, personalisationProvider, statusUpdater);
+
+      spyOn(notifyClient, 'sendEmail');
+      await requestScheduler.scheduleRequests(testResults);
+
+      setTimeout(() => {
+        expect(notifyClient.sendEmail).toHaveBeenCalled();
+        expect(statusUpdater.updateStatus).toHaveBeenCalledWith({
+          applicationReference: 12345672011,
+          outcomePayload: {
+            interface: NOTIFY_INTERFACE,
+            state: ProcessingStatus.ACCEPTED,
+            staff_number: '123456',
+            retry_count: 0,
+            error_message: null,
+          },
+        });
+        done();
+      },         1000);
+    });
+  });
+
+  describe('Terminated test sendLetter handling', () => {
+    it('should not send an email when templateId is empty', async (done) => {
+      const notifyClient: INotifyClient = new NotifyClientStubSuccess();
+      testResults.forEach(result => result.activityCode = '11');
+      const requestScheduler =
+        new RequestScheduler(configAdapter, notifyClient, templateIdProvider, personalisationProvider, statusUpdater);
+
+      spyOn(notifyClient, 'sendLetter');
+      await requestScheduler.scheduleRequests(testResults);
+
+      setTimeout(() => {
+        expect(notifyClient.sendLetter).not.toHaveBeenCalled();
+        expect(statusUpdater.updateStatus).toHaveBeenCalledWith({
+          applicationReference: 12345672011,
+          outcomePayload: {
+            interface: NOTIFY_INTERFACE,
+            state: ProcessingStatus.ACCEPTED,
+            staff_number: '123456',
+            retry_count: 0,
+            error_message: null,
+          },
+        });
+        done();
+      },         1000);
```

Comment:
a few too many spaces between the comma and the 1000

Reply:
without the spaces lint fails